### ISSUE TITLE
Hide Install Button Once PWA is Installed

### ISSUE THREAD

**Issue by richardtaylordawson** [richardtaylordawson/pwa-shields#116]
Hide the install button once the user installs the PWA

**Comment by richardtaylordawson** [comment]
Need to wait on this issue to be merged into pwa-install: https://github.com/pwa-builder/pwa-install/issues/161

**Comment by richardtaylordawson** [comment]
Completed with release v1.10.0
